### PR TITLE
Prettify the README

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,78 +131,81 @@ ORDER BY profit_amount DESC
...
-
-### Tests
-
-Run tests with
-```
-kubectl exec deploy/mev-inspect-deployment -- poetry run pytest --cov=mev_inspect tests
-```
+Postgres tip: Enter `\x` to enter "Explanded display" mode which looks nicer for results with many columns.
 
 ## FAQ
```
Comment (elopio) [inline README.md:136]: I suggest to move the FAQ and most of the usage sections to the github repository wiki.

File: README.md [hunk 2]
```diff
@@ -1,114 +1,128 @@
...
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [helm](https://helm.sh/docs/intro/install/)
 
-Next, install the kubernetes CLI [`kubectl`](https://kubernetes.io/docs/tasks/tools/)
+### Set up
 
-Then, install [helm](https://helm.sh/docs/intro/install/) - helm is a package manager for kubernetes
+Ceate a new cluster with:
 
-Lastly, setup [Tilt](https://docs.tilt.dev/install.html) which manages running and updating kubernetes resources locally
```
Comment (metachris) [inline README.md:30]: Probably should keep the instructions to setup Tilt.

Comment (lukevs) [reply README.md:30]: +1

File: README.md [hunk 3]
```diff
@@ -117,78 +131,81 @@ ORDER BY profit_amount DESC
...
 ```
 kubectl delete pod/postgres-client
 ```
 
-Then start it back up again
+Then start it back up again.
+
+## Maintainers
+
+- [@lukevs](https://github.com/lukevs)
```
[the remark this answers is not in the document]
Comment (lukevs) [reply README.md:192]:
```
@gheise 
@bertmiller 
@taarushv 
```
and you!

File: README.md (hunk 2 above)
Comment (elopio) [reply README.md:30]: On yes, nice catch. Are you ok with moving this link to the dependencies section?

File: README.md [hunk 4]
```diff
@@ -1,114 +1,128 @@
...
 
-First, setup a local kubernetes deployment - we use [Docker](https://www.docker.com/products/docker-desktop) and [kind](https://kind.sigs.k8s.io/docs/user/quick-start)
+### Dependencies
 
-If using kind, create a new cluster with:
-```
-kind create cluster
-```
+- [docker](https://www.docker.com/products/docker-desktop)
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start)
```
[the remark this answers is not in the document]
Comment (elopio) [reply README.md]: I've added a sentence there.

**Pull request merged**